frontend/README.md

```diff
@@ -0,0 +1,52 @@
+# Code.org Frontend
+
+This directory represents the frontend codebase powering Code.org with the exception of the Studio `apps` package. 
```

Comment:
I think this undersells the amount of stuff in `apps` to someone new to our codebase.

"This directory represents a subset of the frontend code powering Code.org (our marketing site and UI component library). The majority of frontend code powering the teacher and student experience (eg, classroom management, learning platform) still lives in the `apps` directory."

Reply:
Great idea, I've updated the README based on your suggestion. Thanks! :)